packages/server/src/events/AutomationEmitter.ts

```diff
@@ -24,7 +15,23 @@ class AutomationEmitter {
     }
   }
 
-  emitRow(eventName: string, appId: string, row: Row, table?: Table) {
+  async getMaxAutomationChain() {
+    const db = context.getAppDB()
+    const appMetadata = await db.get<App>(DocumentType.APP_METADATA)
+    let chainAutomations = appMetadata?.automations?.chainAutomations
+
+    if (chainAutomations === true) {
+      return 5
```

Comment:
Can we make this a constant somewhere - just so the default chaining max is recorded/easy enough to find?